cli/test/smokehouse/frontends/lib.js

```diff
@@ -12,7 +12,7 @@
 
 /* eslint-disable no-console */
 
-import cloneDeep from 'lodash/cloneDeep.js';
+import cloneDeep from 'lodash-es/cloneDeep.js';
```

Comment:
I would expect the advantage of tree-shaking to be that we can import everything from the top-level entry point now. For example:
```suggestion
import {cloneDeep} from 'lodash-es';
```
Is this something you could do?

Reply:
Yes, will make the changes 👌

Reply:
Changes are done